packages/orama/src/methods/insert.ts

```diff
@@ -35,20 +35,35 @@ export async function insert(orama: Orama, doc: Document, language?: string, ski
     const actualType = typeof value
     const expectedType = indexablePropertiesWithTypes[key]
 
+    if (expectedType.endsWith('[]') && Array.isArray(value)) {
+      continue
```

Comment:
I think we want to throw here, like we do on line 43.

Reply:
Done